content/en/tracing/trace_collection/compatibility/cpp.md

```diff
@@ -18,15 +18,13 @@ further_reading:
 
 The C++ Datadog Trace library is open source - view the [GitHub repository][1] for more information.
 
-This library requires C++14 to build, but if you use [dynamic loading][2], then OpenTracing requires [C++11 or later][3].
+This library requires C++17 to build.
```

Comment:
Does this mean we're dropping support for C++14/11? Do we need to notify anyone? 
cc: @tushar-datadog

Reply:
No. It means a C++17 toolchain to compile the library.